SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Notice
+
+The Security Notice for Modrinth projects can be [on our website](https://modrinth.com/legal/security).
```

Comment:
Could be better to put this in a `.github` repository so that we don't have to duplicate it across multiple repos https://github.com/HyacinthBots/.github

Reply:
It should be, as most people who are researching will look for this file